packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -33,7 +33,7 @@ struct TreeTraversal;
  * Bounding Volume Hierarchy.
  */
 template <typename DeviceType>
-struct BVH
+class BVH
```

Comment:
class or struct does not really matter.  Please commit that change in a separate commit.
Say the 1st commit is double brace initialization and this one to be consistent with explicit instantiation.